README.md

```diff
@@ -107,6 +107,12 @@ You can read a [tutorial on how to add a badge][tutorial].
 4. Run `npm start` to start the server.
 5. Open `http://localhost:3000/` to view the frontend.
 
+Shields has experimental support for [Gitpod Beta][gitpod], a pre-configured development
+environment that runs in your browser. To use Gitpod, click the button below and
+sign in with GitHub. Gitpod also offers a browser add-on, though it is not required.
+
+[![Edit with Gitpod](https://img.shields.io/badge/Gitpod-code%20now-blue.svg?logo=gitpod)](https://gitpod.io/#https://github.com/badges/shields)
+
```

Comment:
Could you remove the checked-in logo and replace it with a base64-encoded version in the query string? Sorry, but [we generally do not include logos without a corresponding badge](https://github.com/badges/shields/blob/master/CONTRIBUTING.md#logos). We're trying to put most logos in Simpleicons, though they also have fairly strict inclusion guidelines, we're learning, and I'm not sure Gitpod would be eligible.

In the long run:

- You could think about whether there's an opportunity for a useful status badge for Gitpod, where the badge can hit a REST API and return some useful information. If there's a useful badge we could consider adding it!

- Alternatively – and apologies if this sounds out of turn! – I've another suggestion. Our badges are designed to show status and I think it's a little confusing to use them for actions. If there isn't meaningful status the design intent would be better handled with an action button. It'll do a better job of getting the right people to click it.
- I could also see adding an action button format to Shields if there's interest. Here's some inspiration of one that I think is done well: https://devcenter.heroku.com/articles/heroku-button.

Reply:
> Could you remove the checked-in logo

Logo removed.

> We're trying to put most logos in Simpleicons, though they also have fairly strict inclusion guidelines, we're learning, and I'm not sure Gitpod would be eligible.

I did consider contributing a Gitpod logo to SimpleIcons, but they indeed mandate that your website should be ranked < 500k on Alexa. As of today, [gitpod.io](https://gitpod.io) doesn't quite meet that criterion, but seems to be [almost there](https://www.alexa.com/siteinfo/gitpod.io).

> You could think about whether there's an opportunity for a useful status badge for Gitpod, where the badge can hit a REST API and return some useful information. If there's a useful badge we could consider adding it!

The static badge can indeed be produced using just a Shields URL, which is why I didn't add an actual Gitpod Shields badge implementation. However, and this is even relevant for Shields development, we're about to release a way to pre-build a project's branches and pull requests in the background, so that new workspaces can start almost instantly (without needing to wait for `npm install` for example). Maybe the status of these pre-builds could eventually make sense as a Shields badge (e.g. "in progress" / "ready" / "failed"). 🙂 

> Ha! I think you guys beat me to it:
> 
> https://gitpod.io/button/open-in-gitpod.svg

Indeed, there is also a special SVG icon now as well. It felt strange to add a non-Shields badge to Shields' README, but I agree about the intent being different ("show + link to a status" vs "create a new workspace"), so I'll use that SVG button.

Reply:
> I could also see adding an action button format to Shields if there's interest. Here's some inspiration of one that I think is done well: https://devcenter.heroku.com/articles/heroku-button.

Good idea! I've filed #2824 for this, to see if the idea sounds interesting to others.